### PR TITLE
docs(color): content and image updates

### LINKED ISSUE #1991: [docs] Update Color foundations page
### Description

Requesting minor text and image updates.

[Figma prototype](https://www.figma.com/proto/YhTYnvNpyg4Hgl0WaFDzeI/Color-page-updates?page-id=0%3A1&node-id=1-2&node-type=canvas&viewport=112%2C134%2C1&t=uTOAAzb6JH6oaZxl-1&scaling=scale-down&content-scaling=fixed)

### Links

https://ux.redhat.com/foundations/color/

### Screenshots

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by marionnegp**
## What I did

Closes #1991 

1. Updated Color section based on the ["Color page updates" Figma file](https://www.figma.com/proto/YhTYnvNpyg4Hgl0WaFDzeI/Color-page-updates?page-id=0%3A1&node-id=1-2&node-type=canvas&viewport=112%2C134%2C1&t=uTOAAzb6JH6oaZxl-1&scaling=scale-down&content-scaling=fixed)

## Testing Instructions

1. View the [updated Color page](https://deploy-preview-2020--red-hat-design-system.netlify.app/foundations/color/) in the DP.

### CODE REVIEW

File: docs/foundations/color/usage.md [hunk 1]
```diff
@@ -12,228 +12,179 @@ subnav:
...
-For more bespoke websites or interfaces, other colors may be used as long as 
-color contrast and other accessibility requirements are maintained.
+For more bespoke websites or interfaces, other colors may be used as long as color contrast and other accessibility requirements are maintained.
 
-<rh-alert state="note">
+<rh-alert state="info">
   <h4 slot="header">Helpful tip</h4>
-  <p>Elements like Card and other containers include more colors and can be 
-     placed on top of a canvas.</p>
+  <p>Components like Card and other containers include more colors and can be placed on top of a canvas.</p>
```
Comment (bennypowers) [inline docs/foundations/color/usage.md:75]: use "Elements" instead of "Components". We spent an entire quarter on this, so let's not regress

Comment (marionnegp) [reply docs/foundations/color/usage.md]: Thanks for catching that, @bennypowers! I've replaced any uses of "component(s)."

**Review by coreyvickery — CHANGES_REQUESTED** [review]
@marionnegp Excellent job.

Please re-export the image below the `Red text on backgrounds` heading. The current image has 8px rounded corners. The image in Figma has 3px.

**Comment by marionnegp** [comment]
> @marionnegp Excellent job.
> 
> Please re-export the image below the `Red text on backgrounds` heading. The current image has 8px rounded corners. The image in Figma has 3px.

@coreyvickery, this should be fixed! If it looks correct to you, can you approve this PR?

**Review by coreyvickery — APPROVED** [review]

**Pull request merged**